src/domain/contracts/entities/contract.entity.ts

```diff
@@ -0,0 +1,8 @@
+export class Contract {
+  address: string;
+  name: string;
+  displayName: string;
+  logoUri?: string;
+  contractAbi?: object;
+  trustedForDelegateCall: boolean;
+}
```

Comment:
Following the tx-service model https://github.com/safe-global/safe-transaction-service/blob/master/safe_transaction_service/contracts/models.py#L160 I think logo shouldn't be optional. 
I mean if no logo, the service should return empty string not `null`. 
What do you think?

Reply:
Changed [here.](https://github.com/5afe/safe-client-gateway-nest/commit/7e1b1aa3dca4b4723a3768cb0241b527117a25a3)

Thanks!